_distributor_init.py

```diff
@@ -0,0 +1,26 @@
+""" Distributor init file
+
+Distributors: you can add custom code here to support particular distributions
+of scipy.
+
+For example, this is a good place to put any checks for hardware requirements.
+
+The scipy standard source distribution will not put code in this file, so you
+can safely replace this file with your own version.
+"""
+
+import os
+
+# on Windows SciPy loads important DLLs
+# and the code below aims to alleviate issues with DLL
+# path resolution portability with an absolute path DLL load
+if os.name == 'nt':
+    from ctypes import WinDLL
+    import glob
+    # convention for storing / loading the DLL from
+    # scipy/.libs/, if present
+    libs_path = os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                             '..', '.libs'))
```

Comment:
since this file is in the top level directory, you do not need the `'..'`. Removing that from my local copy let
`python -c"import scipy.linalg"` succeed where with this it fails

Reply:
Tests pass too
<details>

```
================================================ warnings summary =================================================
Python38/Lib/site-packages/scipy/stats/tests/test_binned_statistic.py::TestBinnedStatistic::test_dd_zero_dedges
  c:\pypy_stuff\Python38\lib\site-packages\scipy\stats\_binned_statistic.py:699: RuntimeWarning: divide by zero encountered in log10
    decimal = int(-np.log10(dedges[i].min())) + 6

-- Docs: https://docs.pytest.org/en/latest/warnings.html
====== 19288 passed, 1305 skipped, 10954 deselected, 78 xfailed, 8 xpassed, 1 warnings in 391.50s (0:06:31) =======
```

Reply:
Thanks Matti--I've pushed a commit with you as co-author. We should be much closer overall now to things working.